python/paddle/vision/models/googlenet.py

```diff
@@ -181,7 +189,7 @@ def __init__(self, num_classes=1000, with_pool=True):
             self._drop_o2 = Dropout(p=0.7, mode="downscale_in_infer")
             self._out2 = Linear(1024, num_classes, weight_attr=xavier(1024, 1))
 
-    def forward(self, inputs):
+    def forward(self, inputs: paddle.Tensor) -> list[paddle.Tensor]:
```

Comment:
```suggestion
    def forward(self, inputs: paddle.Tensor) -> tuple[Tensor, Tensor, Tensor]:
```

使用 tuple 吧，输出外的 `[]` 可以删掉，没必要使用 list，当初应该是写错了